course_discovery/apps/course_metadata/data_loaders/api.py

```diff
@@ -953,6 +953,21 @@ def _load_data(self, page):
         response = self._make_request(page)
         self._process_response(response)
 
+    def _add_course_instructors(self, course_instructors, course_run):
+        """
+        Create and add instructors to a course run.
+        """
+        for course_instructor in course_instructors:
+            course_instructor['partner'] = course_run.course.partner
+            instructor, ___ = Person.objects.get_or_create(
```

Comment:
@rehan99000 We haven't considered a case when the information of the instructor is changed. In current scenario we will be adding old instructor (with modified info) as a new instructor which is not a desired approach.

CC: @asadmanzoor93

Reply:
I think we can update `get` params to instructor id and add/update according to that. will update it as soon as panel work is done

Reply:
method updated. now it updates the instructor if instructor is updated as well